### PR TITLE
Add competence document download button

### PR DESCRIPTION
**Pull request by WouterPennings**
- Added download button component for the competence document
 - Discussed with Eline about styling, and followed styling guidde

If you want to see the button, you can add it to the performance dashboard

### CODE REVIEW

File: Epsilon.Host.Frontend/src/components/CompetenceDocumentDownloadButton.vue [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+        <img
+            width="20"
+            height="20"
+            src="@/assets/download-icon.png"
+            alt="download-button-icon" />
+        Download document
+    </a>
+</template>
+
+<style>
```
Comment (Typiqally) [inline Epsilon.Host.Frontend/src/components/CompetenceDocumentDownloadButton.vue]: Use scoped, and maybe SCSS?

File: Epsilon.Host.Frontend/src/components/CompetenceDocumentDownloadButton.vue [hunk 2]
```diff
@@ -0,0 +1,39 @@
+<template>
+    <a
+        class="btn"
+        href="https://localhost:7084/document/word"
+        style="display: flex; justify-content: space-between">
+        <img
+            width="20"
+            height="20"
+            src="@/assets/download-icon.png"
```
Comment (Typiqally) [inline Epsilon.Host.Frontend/src/components/CompetenceDocumentDownloadButton.vue]: If possible, use font awesome

File: Epsilon.Host.Frontend/src/components/CompetenceDocumentDownloadButton.vue [hunk 3]
```diff
@@ -0,0 +1,39 @@
+<template>
+    <a
+        class="btn"
+        href="https://localhost:7084/document/word"
+        style="display: flex; justify-content: space-between">
```
Comment (Typiqally) [inline Epsilon.Host.Frontend/src/components/CompetenceDocumentDownloadButton.vue]: Make class for this

**Review by Typiqally — CHANGES_REQUESTED** [review]

File: Epsilon.Host.Frontend/src/assets/download-icon.png
Comment (NealGeilen) [inline Epsilon.Host.Frontend/src/assets/download-icon.png:1]: Please use a existing font lib

**Review by NealGeilen — CHANGES_REQUESTED** [review]

File: Epsilon.Host.Frontend/src/components/CompetenceDocumentDownloadButton.vue [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+  
+  <script lang="ts" setup>
+    import { ref } from 'vue'
+    import { FontAwesomeIcon } from '@fortawesome/vue-fontawesome'
+    import RoundLoader from "@/components/RoundLoader.vue"
+    import axios from 'axios'
+
+    const isDownloading = ref(false);
+  
+    async function downloadCompetenceDocument() {
```
Comment (NealGeilen) [inline Epsilon.Host.Frontend/src/components/CompetenceDocumentDownloadButton.vue]: Please use the `API` class for requests 
https://github.com/Typiqally/epsilon/blob/develop/Epsilon.Host.Frontend/src/logic/Api.ts

**Review by NealGeilen — CHANGES_REQUESTED** [review]

**Comment by NealGeilen** [comment]
Also, please fix the GitHub action error

**Review by NealGeilen — APPROVED** [review]

**Review by Typiqally — APPROVED** [review]
Very nice button, I like it.

**Pull request merged**